driver-sync/src/test/functional/com/mongodb/client/unified/ContextElement.java

```diff
@@ -83,21 +87,41 @@ public String toString() {
         }
     }
 
-    private static class OperationContextElement extends ContextElement {
+    private static class StartedOperationContextElement extends ContextElement {
+        private final BsonDocument operation;
+        private final int index;
+
+        StartedOperationContextElement(final BsonDocument operation, final int index) {
+            this.operation = operation;
+            this.index = index;
+        }
+
+        public String toString() {
+            return "Started Operation Result Context: " + "\n"
```

Comment:
I do not understand what "Started Operation Result Context" is. Maybe "Started Operation Context" is less confusing given that there is no result?

Reply:
Done

Reply:
I think part of what makes this confusing is that "Started" (and "Completed" below) are used here as adjectives modifying "Operation", rather that verbs.